variables.tf

```diff
@@ -437,8 +437,11 @@ variable "attach_policy_statements" {
 
 variable "trusted_entities" {
   description = "Lambda Function additional trusted entities for assuming roles (trust relationship)"
-  type        = list(string)
-  default     = []
+  type = list(object({
```

Comment:
Could you change this to `list(any)` and use `try()` to make this PR backward compatible and support previous values also.

I think it should be rather straightforward. Let me know if you have questions.

Reply:
It’s a very good idea, I will try this and keep you informed on this, thank you very much !